eslint.config.mjs

```diff
@@ -12,27 +12,44 @@ export default tseslint.config(
             parserOptions: {
                 projectService: true,
                 // @ts-expect-error import.meta
-                tsconfigRootDir: import.meta.dirname,
-            },
+                tsconfigRootDir: import.meta.dirname
+            }
         },
         plugins: {
-            "@stylistic": stylistic,
+            "@stylistic": stylistic
         },
         rules: {
             "@stylistic/brace-style": ["error", "1tbs", { allowSingleLine: true }],
             "@stylistic/indent": ["error", 4],
             "@stylistic/quotes": ["error", "double"],
             "@stylistic/semi": ["error", "always"],
             "@typescript-eslint/no-base-to-string": "off",
+            "@typescript-eslint/no-unsafe-assignment": "off",
```

Comment:
Why did you ignore those rules, they are necessary to keep the code safe and clean